core/src/consensus.rs

```diff
@@ -986,24 +1013,24 @@ impl Tower {
     ) -> bool {
         let mut vote_state = self.vote_state.clone();
         process_slot_vote_unchecked(&mut vote_state, slot);
-        let vote = vote_state.nth_recent_vote(self.threshold_depth);
-        if let Some(vote) = vote {
-            if let Some(fork_stake) = voted_stakes.get(&vote.slot()) {
-                let lockout = *fork_stake as f64 / total_stake as f64;
+        let lockout = vote_state.nth_recent_lockout(self.threshold_depth);
```

Comment:
try to avoid making cosmetic changes like this rename in the same PR as logical changes.  they make review more difficult from both sides

Reply:
I understand the point but this is a minor change and pretty easy to follow if you try.  Additionally, the lack of consistency in naming makes the code altogether harder to read.  The change of name to nth_recent_lockout is meant to ensure that the logical purpose of the code change in this commit can be followed.

Reply:
By the way if there is any chance of this being a blocking issue, please let me know and I'll move that rename into a separate change.